test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,172 @@
+log-path=$PGBACKREST_TEST_DIR/logs/$TEST_NAME
+start-fast=y
+fork=GPDB
+EOF
+
+# Initializing pgbackrest for GPDB
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i stanza-create
+done
+
+# Configuring WAL archiving command
+gpconfig -c archive_mode -v on
+
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" \
+$PGBACKREST_BIN/pgbackrest --config $PGBACKREST_TEST_DIR/pgbackrest.conf \
+--stanza=seg%c archive-push %p'" --skipvalidation
+
+gpstop -ar
+
+# pgbackrest health check
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i check
+done
+
+# The test scenario starts here
+# Creating initial dataset
+export PGDATABASE=gpdb_pitr_database
+createdb
+psql -c "CREATE TABLE t1 AS SELECT id, 'text'||id AS text FROM generate_series(1,30) id DISTRIBUTED BY (id);"
+
+# Creating full backup on master and seg0
+for i in -1 0
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+# Checking the presence of first backup
+function check_backup(){
+    segment_backup_dir=$PGBACKREST_TEST_DIR/$TEST_NAME/backup/seg$1
+    current_date=$(date +%Y%m%d)
+    test -n "$(find "$segment_backup_dir" -maxdepth 1 -type d -name "${current_date}-??????F" -not -empty)"
+}
+for i in -1 0
+do 
+   check_backup $i
+done
+
+# Creating additional table
+psql -c "CREATE TABLE t2 AS SELECT id, 'text'||id AS text FROM generate_series(1,30) id DISTRIBUTED BY (id);"
+
+psql -c "SELECT * FROM t2 ORDER BY id;" \
+-o "$PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out"
+
+# Filling the first table with more data at seg0 after the first backup
+psql -c "INSERT INTO t1 SELECT 3, 'text'||i FROM generate_series(1,10) i;"
+
+psql -c "SELECT * FROM t1 ORDER BY id;" \
+-o "$PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_original.out"
+
+# Creating full backup on seg1 and seg2
+for i in 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+# Checking the presence of second backup
+for i in 1 2
+do 
+   check_backup $i
+done
+
+# Creating a distributed restore point..."
+psql -c "create extension gp_pitr;"
+psql -c "select gp_create_restore_point('test_pitr');"
+psql -c "select gp_switch_wal();"
+
+# Simulating disaster
+psql -c "drop table t1;"
+psql -c "truncate table t2;"
+
+gpstop -a
+rm -rf "${MASTER:?}/"* "${PRIMARY1:?}/"* "${PRIMARY2:?}/"* "${PRIMARY3:?}/"*
+rm -rf "${MIRROR1:?}/"* "${MIRROR2:?}/"* "${MIRROR3:?}/"* "$DATADIR/standby/"*
+
+# Restoring cluster
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest --config \
```

Comment:
Why PGOPTIONS is needed here? The cluster is stopped, so pgbackrest cannot connect to it.